### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by Lon1st** [joshdjuric/Skatter#3]
Adding random opacity feature would be great!

**Comment by joshdjuric** [comment]
@Lon1st Yeah not a bad shout - I'm rather clumsily working on a native panel to have settings on rather than being bombarded with consecutive dialogues. The concept of skattering opacity is rather straight forward - I'm hoping to release it with the panel.
